.github/workflows/publish-extensions.yml

```diff
@@ -43,19 +54,28 @@ jobs:
       - name: Sign Production Firefox version
         continue-on-error: true
         run: yarn web-ext sign --channel=unlisted
+        # run: yarn web-ext sign --channel=listed
         env:
-          WEB_EXT_API_KEY: ${{ secrets.WEB_EXT_API_KEY }}
-          WEB_EXT_API_SECRET: ${{ secrets.WEB_EXT_API_SECRET }}
-      - uses: actions/upload-artifact@v2
-        name: Upload Artifact - Firefox Add-On XPI
-        with:
-          name: connect-addon
-          path: web-ext-artifacts/*.xpi
+          WEB_EXT_API_KEY: ${{ secrets.FIREFOX_API_KEY }}
+          WEB_EXT_API_SECRET: ${{ secrets.FIREFOX_API_SECRET }}
```

Comment:
I'm assuming `WEB_EXT_API_KEY` and `FIREFOX_API_KEY` are the same? Because `WEB_EXT_API_KEY` is still used in the pull_request workflow, and seems to be working.

Reply:
I'm not sure, I tried reaching out in Discord to see if you knew the values of the `WEB_EXT_API_KEY` and `WEB_EXT_API_SECRET`, but never got a response. I ended up creating API creds from the firefox store using the `firefox` account. If that's what `web-ext` uses, then it can use the new secrets I created.